### DISCUSSION TITLE
Carplay-cast Discussions

### DISCUSSION THREAD

**Discussion by EthanArbuckle** [EthanArbuckle/carplay-cast#23] [Q&A]
Use Discussions tab to ask questions about tweak usage or installation problems.

**Comment by crowe769** [comment]
> thank you sorry still learning

**Comment by XqqsMe** [comment]
Sorry as well. I’m really wanting the tweak but think I may need jailbroken first. Is that correct?

**Reply by EthanArbuckle** [reply]
Yes it requires a jailbroken device on iOS 14.0+

**Comment by jly26** [comment]
Hi will there be a 13.5 spupport??

**Reply by EthanArbuckle** [reply]
It’s not planned, but other devs have shown interest in backporting

**Reply by jly26** [reply]
Thanks.. someone backport your tweaks on 13.5 https://github.com/leftyfl1p/carplay-cast . I tried this and works but with bunch of issues. One main issue was when an active apps (youtube etc) is diplaying in carplay , my iphone 13.5 dsiplay  must be active too, and when  the display  sleeps , black display will be on carplay .Most people are on 13.5 and I think I can help by sending you some error logs

**Comment by XqqsMe** [comment]
I have the iPhone 12 Pro Max. Running iOS 14.4. Will the tweak work with it now? If so, could someone walk me thru how to install please?
